### PR TITLE
feat(ui): introduce splash screen

### LINKED ISSUE #49: Add loading screen with own bootstrap
maybe with a splash screen or even as PWA

### PR DESCRIPTION
**Pull request by marianfoo**
Fixes #49

### CODE REVIEW

File: packages/ui/src/index.html [hunk 1]
```diff
@@ -9,6 +9,32 @@
...
+
+		<style>
+			html, body {
+				height: 100%;
+			}
+			body.splash {
+				display: inline-block;
+				background-position: center;
+				background-repeat: no-repeat;
+				background-size: contain;
```
Comment (petermuessig) [inline packages/ui/src/index.html:22]: This should do the trick now to keep the size

**Review by petermuessig — APPROVED** [review]
@marianfoo - please take a look again - should work now - incl. light, dark mode using a different UI5 logo: dark => orange, light => blue.

File: packages/ui/src/index.html (hunk 1 above)
Comment (marianfoo) [reply packages/ui/src/index.html:22]: works!

**Comment by marianfoo** [comment]
perfect! looks good
i added a background color because the dark image is loaded with white background in short period at the beginning
you can squash/merge if you like

**Pull request merged**